tests/test_validators.py

```diff
@@ -8,38 +8,68 @@
 
 class TestValidators(object):
     def test_valid(self):
+        valid_vatin = VATIN(VALID_VIES_COUNTRY_CODE, VALID_VIES_NUMBER)
         validator = VATINValidator()
-        vatin = VATIN(VALID_VIES_COUNTRY_CODE, VALID_VIES_NUMBER)
-        validator(vatin)
+        validator(valid_vatin)
 
         validator = VATINValidator(verify=False, validate=True)
-        vatin = VATIN(VALID_VIES_COUNTRY_CODE, VALID_VIES_NUMBER)
-        validator(vatin)
+        validator(valid_vatin)
 
         validator = VATINValidator(verify=True, validate=True)
-        vatin = VATIN(VALID_VIES_COUNTRY_CODE, VALID_VIES_NUMBER)
-        validator(vatin)
+        validator(valid_vatin)
 
     def test_no_check_exception(self):
         with pytest.raises(ValueError):
             VATINValidator(verify=False, validate=False)
 
     def test_invalid(self):
+        invalid_number_vatin = VATIN(VALID_VIES_COUNTRY_CODE, '12345678')
+        invalid_country_vatin = VATIN('XX', VALID_VIES_NUMBER)
+        validator = VATINValidator()
+
+        with pytest.raises(ValidationError):
+            validator(invalid_country_vatin)
+
+        validator = VATINValidator(verify=False, validate=True)
+        with pytest.raises(ValidationError):
+            validator(invalid_number_vatin)
+
+        validator = VATINValidator(verify=True, validate=True)
+        with pytest.raises(ValidationError):
+            validator(invalid_country_vatin)
+
+        with pytest.raises(ValidationError):
+            validator(invalid_number_vatin)
+
+
+class TestValidatorsWithoutVATINObject(object):
+    """Validate raw values without VATIN object from model field."""
+    def test_valid(self):
+        valid_vatin = ''.join([VALID_VIES_COUNTRY_CODE, VALID_VIES_NUMBER])
+        validator = VATINValidator()
+        validator(valid_vatin)
+
+        validator = VATINValidator(verify=False, validate=True)
+        validator(valid_vatin)
+
+        validator = VATINValidator(verify=True, validate=True)
+        validator(valid_vatin)
+
+    def test_invalid(self):
```

Comment:
you can [parametrise a test](http://docs.pytest.org/en/latest/parametrize.html) so it runs with both a string and and VATIN.
I'd prefer tho, if you just add a test for the new method. And leave the rest as is.